wsmaster/che-core-api-workspace/src/main/resources/schema/devfile.json

```diff
@@ -279,7 +279,11 @@
                       "{\"java.home\": \"/home/user/jdk11\", \"java.jdt.ls.vmargs\": \"-Xmx1G\"}"
                     ],
                     "additionalProperties": {
-                      "type": "string"
+                      "type": [
+                        "boolean",
+                        "string",
+                        "integer"
```

Comment:
Why not a `number` here instead of just `integer`?

Reply:
yup will fix